### DISCUSSION TITLE
Can't build any project from the example... but I can build other next Project from outside vercel/ next-create-app

### DISCUSSION THREAD

**Discussion by sarahBuisson** [vercel/next.js#52882] [Help]
### Verify canary release

- [X] I verified that the issue exists in the latest Next.js canary release

### Provide environment information

```bash
Operating System:
      Platform: win32
      Arch: ia32
      Version: Windows 10 Home
    Binaries:
      Node: 18.15.0
      npm: N/A
      Yarn: N/A
      pnpm: N/A
    Relevant Packages:
      next: 13.4.8-canary.14
      eslint-config-next: N/A
      react: 18.2.0
      react-dom: 18.2.0
      typescript: N/A
    Next.js Config:
      react: 18.2.0
      react-dom: 18.2.0
      typescript: 5.1.6
    Next.js Config:
      output: N/A
```

### Which example does this report relate to?

with-typescript

### How are you deploying your application? (if relevant)

next build; next start

### Describe the Bug

Sorry for the bad english

TLDR : no (builded) file in .next dir when I do 'next build'

when I do npm run build:
I have only those 3 line of log

- info Downloading WASM swc package...
- info Using wasm build of next-swc
- info Creating an optimized production build .

Then it stop, and I have only 2 file in the .next dir:
package.json
routes-manifest.json

Obviously the project don't start.

I have the same issue with all the next project that come from vercel/example, next-learn, etc.

I don't have nextConfig nor appDir defined, but when I have, I have the same issue ( in the appDir)

I have only this issue with project who either come from vercel git repo or are build with create-next-app

**BUT** 

The nextJs ts project https://github.com/jean-script/SujeitoPizzaria_frontend
work fine. 

Now is the fun part : If I use the package.json of SujeitoPizzaria_frontend in a create-next-app typescript project, it work perfectly.

Can you investigate and fix it?

### Expected Behavior

more file in the .next directory

Also, more log to help investigate. At least a log saying "congrat, the build work fine/ sorry the build fail" at the end.

### To Reproduce

 npx create-next-app --example with-typescript nextjs-app3

cd nextjs-app3

npm run build

cd .next
